### PR TITLE
Swift 3 Syntax for Multi-clause conditions

### PR DESCRIPTION
**Pull request by frankieshakes**
- Updated multi-clause conditions to use Swift 3 syntax
- Changed data.length to data.count for byte length example.

### CODE REVIEW

**Comment by AliSoftware** [comment]
Cool, thx!

I think I'll start to use the Swift 3.0 badge on top of the articles which have been revised for Swift 3, so that people reading my old article will know which ones have been converted to new syntax and which ones haven't yet. Could you add that badge at the top?

Also as you suggested in the comment on my blog post, I think it's worth converting it to Swift 3 conventions as well (lowercase enums and so on).

File: _posts/2016-05-16-pattern-matching-4.md [hunk 1]
```diff
@@ -57,10 +57,10 @@ switch m {
 
 ## if case let where
 
-We can combine the `if case let` with a `where` clause of course:
+We can combine the `if let case` with a comma (`,`) -- where each condition is separated by `,` -- to create a multi-clause condition:
```
Comment (AliSoftware) [inline _posts/2016-05-16-pattern-matching-4.md]: Why the change from `case let` to `let case` here?

**Review by AliSoftware — CHANGES_REQUESTED** [review]

File: _posts/2016-05-16-pattern-matching-4.md (hunk 1 above)
Comment (frankieshakes) [reply _posts/2016-05-16-pattern-matching-4.md]: Ooops. That's my mistake. It's still `if case let` ;) 

I'll update that with my next batch of changes.

**Review by AliSoftware — APPROVED** [review]

**Comment by AliSoftware** [comment]
Thanks again for porting these posts to Swift 3! 👍 👌

**Pull request merged**